Antidote/ActiveCallViewController.h

```diff
@@ -23,7 +23,16 @@
 
 - (void)activeCallDeclineIncomingCallButtonPressed:(ActiveCallViewController *)controller;
 - (void)activeCallAnswerIncomingCallButtonPressed:(ActiveCallViewController *)controller;
-- (void)activeCallPausedCallSelectedAtIndex:(NSUInteger)index controller:(ActiveCallViewController *)controller;
+
+@end
+
+@protocol ActiveCallViewControllerDataSource <NSObject>
+
+- (NSInteger)activeCallControllerNumberOfPausedCalls:(ActiveCallViewController *)controller;
+- (NSString *)activeCallController:(ActiveCallViewController *)controller pausedCallerNicknameForCallAtIndex:(NSIndexPath *)indexPath;
+- (NSTimeInterval)activeCallController:(ActiveCallViewController *)controller pauseTimeDurationForCallAtIndex:(NSIndexPath *)indexPath;
+- (void)activeCallController:(ActiveCallViewController *)controller resumePausedCallSelectedAtIndex:(NSIndexPath *)indexPath;
+- (void)activeCallController:(ActiveCallViewController *)controller endPausedCallSelectedAtIndex:(NSIndexPath *)indexPath;
```

Comment:
Not sure what you mean? The indexPath is needed because the `RBQFetchedResultsController` has a method for `objectAtIndexPath:indexPath` which I use in those methods.

`OCTCall *call = [self.allPausedCallsController objectAtIndexPath:indexPath];`